getting-started/data-scientists.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Data scientists
+description: Get started with Coder as a data scientist.
+---
+
+This article will walk you through the process of getting started with a Coder
+workspace capable of supporting data science projects. You'll learn how to:
+
+- Connect Coder to your Git provider;
+- Create a workspace with Jupyter and other data science packages present;
+- Add a sample project to your workspace, specifically
+  [one in Jupyter notebook form using IMDB movie data](https://github.com/khorne3/data-science-imdb-sample)
```

Comment:
seems this should also end with a semicolon, or remove all of the trailing semicolons (that would be my preference, personally, but I suppose it depends on whether you want to end the list with a full stop?)

The benefit of avoiding punctuation is that it makes the list easier to reorder